lib/queries/useAuthoritiesBrowse/useAuthoritiesBrowse.js

```diff
@@ -73,62 +74,48 @@ const useBrowseRequest = ({
     },
   );
 
-  const firstResult = isFetching ? null : get(data, 'items[0].headingRef');
-  const lastResult = isFetching ? null : get(data, `items[${data?.items?.length - 1}].headingRef`);
-
   return ({
     isFetched,
     isFetching,
     data,
-    firstResult,
-    lastResult,
     query,
   });
 };
 
-const useBrowserPaging = (initialQuery, initialIndex) => {
-  const _initialQuery = initialQuery.replaceAll('"', '\\"');
-  const [page, setPage] = useState(0);
+const useBrowserPaging = (searchQuery, searchIndex, browsePageQuery, setBrowsePageQuery, navigationSegmentValue) => {
+  const _searchQuery = searchQuery.replaceAll('"', '\\"');
 
-  const getMainRequestSearch = (newQuery, newPage) => {
-    let newMainRequestSearch = [];
+  const buildPageQuery = (newQuery, direction) => {
     const _newQuery = newQuery.replaceAll('"', '\\"');
 
-    if (newPage < page) { // requested prev page
-      newMainRequestSearch = [`headingRef<"${_newQuery}"`];
-    } else if (newPage > page) { // requested next page
-      newMainRequestSearch = [`headingRef>"${_newQuery}"`];
-    }
-
-    if (newPage === 0) {
```

Comment:
We need to know if current page is 0 to get results with anchor
With this change if you go to previous page, and back to next you won't see an anchor

https://github.com/folio-org/stripes-authority-components/assets/19309423/ee43ad93-0061-4375-ace2-41cc212598e4

Reply:
Thanks for that, I've fixed it.